C/sort/mergesort_in_O(1)_space_complexity.c

```diff
@@ -0,0 +1,114 @@
+// algorithm for merging two sorted array in O(1) space complexity and in
+// Nlog(N) + Mlog(M) time complexity
+#include <stdio.h>
+#include <stdlib.h>
+int i, j;
+
+void mergesort(int a[], int b[], int n, int m) {
+  int min;
```

Comment:
@ag278 You have imported `#include <stdlib.h>` so you can use `int minimum = min(n, m)`.